### ISSUE TITLE
Submodules for Downstream Contributors

### ISSUE THREAD

**Issue by swizzley** [capitalone/Hygieia#1305]
Could someone please break this Repository up into git submodules so that folks can contribute to individual components without exposing our private forks?

**Comment by mrpudn** [comment]
Why submodules? They can be a real pain to work with. What's the pain point you're experiencing?

**Comment by tabladrum** [comment]
@mrpudn see the gitter thread.

**Comment by mrpudn** [comment]
Yup, just went through, and I think I'm aligned, although it might be a bit of an overhaul

**Comment by swizzley** [comment]
Basically because we have code, (not limited to collectors) that is for internal APIs, but we would like to contribute to your code base directly from git without having to do any manual trickery or tom-foolery to keep our private code private. =-)

**Comment by mrpudn** [comment]
Gotcha, that makes sense :)
